bin/inference/pycbc_inference_start_from_samples

```diff
@@ -0,0 +1,49 @@
+#!/bin/env python
+""" Convert inference file to parallel temperred compatible start format
+"""
+
+import argparse
+import numpy
+import h5py
+from numpy.random import choice
+from pycbc.inference.io import loadfile
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input-file')
+parser.add_argument('--output-file')
+parser.add_argument('--sampler', default='emcee_pt_file',
+    help="The output sampler type, if none, we assume emcee_pt")
```

Comment:
Make the default `emcee_pt`; i.e., drop the `_file` bit. You can get the file type name using the sampler class (see comment below).